app/src/main/java/net/aiscope/gdd_app/model/SampleModels.kt

```diff
@@ -31,6 +33,17 @@ data class Sample(
     fun nextImageName(): String = "${id}_image_${captures.completedCaptureCount()}"
 
     fun nextMaskName(): String = "${id}_mask_${captures.completedCaptureCount()}"
+
+    fun buildSampleCollection() : SampleCollection {
```

Comment:
you can one line it doing fun buildSampleCollection() = SampleCollection(
            createdOn = Timestamp(createdOn.time),
            uploadedBy = FirestoreUtil.FirestoreUtil.firestore?.
            collection("microscopist")?.document(microscopist),
            location = id,
            numberOfImages = captures.completedCaptureCount()
        )